.dockerignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
Need to have all the dot files

Reply:
and do check for other files not required

Reply:
Made changes in the new PR from proper branch:

https://github.com/jagatjeevan/nextjs-setup/pull/3